dts/bindings/binding-template.yaml

```diff
@@ -56,15 +56,23 @@ sub-node:
 #
 #   <property name>:
 #     category: <required | optional>
-#     type: <string | int | boolean | array | uint8-array | string-array | compound>
+#     type: <string | int | boolean | array | uint8-array | string-array | phandle | compound>
```

Comment:
Please also regard multiple phandles as phandle-array for the pinctrl-x property